R/add_miQC.R

```diff
@@ -12,7 +12,7 @@
 #'
 #' @export
 #'
-add_miQC <- function(sce, seed = NULL){
+add_miQC <- function(sce, posterior_cutoff = 0.7, seed = NULL){
```

Comment:
```suggestion
add_miQC <- function(sce, posterior_cutoff = 0.75, seed = NULL){
```
I believe this should be 0.75 if we are sticking with the default for the filtering function in miQC. I don't think we really have rationale to not use the default right now. https://bioconductor.org/packages/devel/bioc/vignettes/miQC/inst/doc/miQC.html#33_Extras